samples/matter/window_covering/Kconfig.sysbuild

```diff
@@ -5,6 +5,10 @@
 #
 
 source "${ZEPHYR_BASE}/share/sysbuild/Kconfig"
+source "${ZEPHYR_NRF_MODULE_DIR}/boards/shields/nrf7002ek/Kconfig.shield"
 
 config NRF_DEFAULT_MULTIPROTOCOL
-	default y
+	default y if ($(BOARD) = "nrf5340dk_nrf5340_cpuapp") && !SHIELD_NRF7002EK
+
+config NRF_DEFAULT_BLUETOOTH
+	default y if ($(BOARD) = "nrf7002dk_nrf5340_cpuapp") || SHIELD_NRF7002EK
```

Comment:
We do not support Wi-Fi for window covering.

Reply:
This change will be reverted anyway.